lib/routes/skeb/saerch.ts

```diff
@@ -0,0 +1,77 @@
+import { Data, DataItem, Route } from '@/types';
+import cache from '@/utils/cache';
+import ofetch from '@/utils/ofetch';
+import { processWork, baseUrl } from './utils';
+import InvalidParameterError from '@/errors/types/invalid-parameter';
+
+export const route: Route = {
+    path: '/search/:keyword',
+    categories: ['picture'],
+    example: '/search/初音ミク',
+    parameters: { keyword: 'Search keyword' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: 'Search Results',
+    maintainers: ['SnowAgar25'],
+    handler,
+    description: 'Get the search results for works on Skeb',
+};
+
+async function handler(ctx): Promise<Data> {
+    const keyword = ctx.req.param('keyword');
+
+    if (!keyword) {
+        throw new InvalidParameterError('Invalid search keyword');
+    }
```

Comment:
This will always be skipped since https://github.com/DIYgod/RSSHub/blob/4028d798f71ecd280043b8f869f3cd5a25c9e2b0/lib/routes/skeb/saerch.ts#L8 defines `keyword` as a compulsory parameter.